mainmemory.cpp

```diff
@@ -241,7 +241,7 @@ void MainMemory::cellDataChanged(QTableWidgetItem *item)
 {
     // disconnect this signal so that modifying the text of the column next to it doesn't fire this signal; reconnect at the end
     disconnect(ui->tableWidget, SIGNAL(itemChanged(QTableWidgetItem*)), this, SLOT(cellDataChanged(QTableWidgetItem*)));
-
+    disconnect(dataSection,&CPUDataSection::memoryChanged,this,&MainMemory::onMemoryValueChanged);
```

Comment:
I'm pretty sure that this should instead be:
`disconnect(dataSection, SIGNAL(memoryChanged(quint16,quint8,quint8)), this, SLOT(onMemoryValueChanged(quint16,quint8,quint8)));`
...as that uses the macros supplied by Qt.

Line 272 similar.